# Definition of a box model for the boxpick component

# Required
string type

# Required
RangeRectangle rectangle
